tests/unit/anchore_engine/services/policy_engine/policy/gates/test_dockerfile.py

```diff
@@ -0,0 +1,297 @@
+import pytest
+from anchore_engine.services.policy_engine.engine.policy.gates import dockerfile
+from anchore_engine.services.policy_engine.engine.policy.gate import ExecutionContext
+from anchore_engine.db.entities.policy_engine import Image
+
+image_id = "1"
+user = "admin"
+absent_modes = ["guessed", None]
+dockerfile_contents = """FROM ubuntu:18.04
+USER admin
+COPY . /app
+RUN make /app
+CMD python /app/app.py
+EXPOSE 3000
+"""
+
+
+@pytest.fixture()
+def image():
```

Comment:
global: naming. I've seen methods named like this before for builders and such, but they're more for adding attributes or getting attributes (i.e. auto value in Java). It'd be a bit more clear to subsequent readers if this was `get_image` or `construct_image` or `create_image`

Reply:
Thanks! This got me thinking and I was actually able to refactor down to a single test method